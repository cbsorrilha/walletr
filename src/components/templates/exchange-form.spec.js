import React from 'react';
import { render } from '@testing-library/react';
import ExchangeForm from './exchange-form';

describe('Test the exchange-form component', () => {
  it('should match the snapshots', () => {
    const component = render(<ExchangeForm />);
    expect(component).toMatchSnapshot();
  });
});
