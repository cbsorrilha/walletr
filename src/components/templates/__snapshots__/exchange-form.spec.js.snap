// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the exchange-form component should match the snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-ifAKCX jnZHkZ"
      >
        <div
          class="sc-EHOje fxFbyW"
          width="100%"
        >
          <form>
            <div
              class="sc-EHOje cAUkgB"
            >
              <div
                class="sc-ifAKCX fndciJ"
              >
                <div
                  class="sc-EHOje cAUkgB"
                >
                  <p
                    class="sc-bdVaJa cPeglz"
                  >
                    Saldo: 
                    undefined undefined
                  </p>
                </div>
                <div
                  class="sc-EHOje duAuhe"
                >
                  <a
                    class="sc-bwzfXH jkSlsb"
                  >
                    Tudo
                  </a>
                </div>
              </div>
            </div>
            <div
              class="sc-gzVnrw eZFjIs"
              height="auto"
            >
              <div
                class="sc-htoDjs IgGWb"
              >
                <p
                  class="sc-bdVaJa ejRlfi"
                />
              </div>
              <div
                class="sc-htoDjs bScaTT"
              >
                <div
                  class="sc-ifAKCX idlhrb"
                  direction="column"
                >
                  <div
                    class="sc-EHOje cAUkgB"
                  >
                    <input
                      class="sc-bZQynM DRnwO"
                      name="amount-undefined"
                      type="number"
                      value=""
                      width="13.5rem"
                    />
                  </div>
                  <div
                    class="sc-EHOje fkfkya"
                    height=".5rem"
                  />
                </div>
              </div>
              <div
                class="sc-htoDjs bScaTT"
              >
                <div
                  class="sc-EHOje dllnsp"
                >
                  <p
                    class="sc-bdVaJa jktShC"
                  >
                    ➡
                  </p>
                </div>
              </div>
              <div
                class="sc-htoDjs dDKvkC"
              >
                <p
                  class="sc-bdVaJa ejRlfi"
                >
                   
                </p>
              </div>
            </div>
            <button
              class="sc-bxivhb ewxeyt"
              type="submit"
            />
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-ifAKCX jnZHkZ"
    >
      <div
        class="sc-EHOje fxFbyW"
        width="100%"
      >
        <form>
          <div
            class="sc-EHOje cAUkgB"
          >
            <div
              class="sc-ifAKCX fndciJ"
            >
              <div
                class="sc-EHOje cAUkgB"
              >
                <p
                  class="sc-bdVaJa cPeglz"
                >
                  Saldo: 
                  undefined undefined
                </p>
              </div>
              <div
                class="sc-EHOje duAuhe"
              >
                <a
                  class="sc-bwzfXH jkSlsb"
                >
                  Tudo
                </a>
              </div>
            </div>
          </div>
          <div
            class="sc-gzVnrw eZFjIs"
            height="auto"
          >
            <div
              class="sc-htoDjs IgGWb"
            >
              <p
                class="sc-bdVaJa ejRlfi"
              />
            </div>
            <div
              class="sc-htoDjs bScaTT"
            >
              <div
                class="sc-ifAKCX idlhrb"
                direction="column"
              >
                <div
                  class="sc-EHOje cAUkgB"
                >
                  <input
                    class="sc-bZQynM DRnwO"
                    name="amount-undefined"
                    type="number"
                    value=""
                    width="13.5rem"
                  />
                </div>
                <div
                  class="sc-EHOje fkfkya"
                  height=".5rem"
                />
              </div>
            </div>
            <div
              class="sc-htoDjs bScaTT"
            >
              <div
                class="sc-EHOje dllnsp"
              >
                <p
                  class="sc-bdVaJa jktShC"
                >
                  ➡
                </p>
              </div>
            </div>
            <div
              class="sc-htoDjs dDKvkC"
            >
              <p
                class="sc-bdVaJa ejRlfi"
              >
                 
              </p>
            </div>
          </div>
          <button
            class="sc-bxivhb ewxeyt"
            type="submit"
          />
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
